src/render/vulkan/SDL_render_vulkan.c

```diff
@@ -4024,13 +4025,16 @@ static int VULKAN_SetVSync(SDL_Renderer *renderer, const int vsync)
 {
     VULKAN_RenderData *rendererData = (VULKAN_RenderData *)renderer->driverdata;
 
-    Uint32 prevFlags = renderer->info.flags;
-    if (vsync) {
-        renderer->info.flags |= SDL_RENDERER_PRESENTVSYNC;
-    } else {
-        renderer->info.flags &= ~SDL_RENDERER_PRESENTVSYNC;
+    switch (vsync) {
+    case 0:
+    case 1:
+        /* Supported */
+        break;
+    default:
+        return SDL_Unsupported();
```

Comment:
A vsync value of -1 in SDL would be VK_PRESENT_MODE_FIFO_RELAXED_KHR in Vulkan. I haven't parsed through all of `VULKAN_CreateSwapChain` but that would be where this would get added. If that's out of scope for this PR, we should open an issue so we remember to deal with it later.

Reply:
This is in, as a separate commit.